Failed to run arch-test on arm64 machine

Prebuilt binary of Sail doesn't exist.
I was running test on Macbook M1, commit `164da62b64fa050569b22910ea207525884802ba`.
```
$make arch-test
  CC    build/map.o
  CC    build/utils.o
  CC    build/decode.o
  CC    build/io.o
  CC    build/syscall.o
  CC    build/emulate.o
  CC    build/riscv.o
  CC    build/elf.o
  CC    build/cache.o
  CC    build/mpool.o
  CC    build/main.o
  LD    build/rv32emu
Fetching SHA-1 of prebuilt binaries ... [OK]
Checking SHA-1 of prebuilt binaries ... [OK]
git submodule update --init tests/
cp: build/sail_cSim/riscv_sim_RV32: No such file or directory
make: *** [arch-test] Error 1
```

--- a/README.md
+++ b/README.md
@@ -196,9 +196,14 @@ $ python3 -m pip install git+https://github.com/riscv/riscof
 [RISC-V GNU Compiler Toolchain](https://github.com/riscv-collab/riscv-gnu-toolchain) should be prepared in advance.
 You can obtain prebuilt GNU toolchain for `riscv32-elf` from the [Automated Nightly Release](https://github.com/riscv-collab/riscv-gnu-toolchain/releases).
 If `ENABLE_SYSTEM=1` was previously set, run `make distclean` before proceeding.
-Then, run the following command:
+Then, run the following command to run all tests:
 ```shell
-$ make ENABLE_ARCH_TEST=1 arch-test
+$ .ci/riscv-tests.sh
+```
+
+Or run it with the configuration you want:
+```shell
+$ make arch-test ENABLE_ARCH_TEST=1 ENABLE_FULL4G=1 <your-config>
 ```
 
 For macOS users, installing `sdiff` might be required:

